lib/qa/authorities/web_service_base.rb

```diff
@@ -11,9 +11,15 @@ def get_json(url)
     end
 
     def response(url)
-      Faraday.get(url) do |req|
-        req.headers['Accept'] = 'application/json'
+      uri = URI(url)
+      conn = Faraday.new uri.scheme+"://"+uri.host
+      conn.options.params_encoder = Faraday::FlatParamsEncoder
```

Comment:
Is it possible to limit this just to `id.loc.gov`? I'd assume other providers expect (or, at least, support) Faraday's default behavior.